pkg/util/kubeconfig_test.go

```diff
@@ -394,6 +394,107 @@ func TestDeleteKubeConfigContext(t *testing.T) {
 	}
 }
 
+func TestSetCurrentContext(t *testing.T) {
+	contextName := "minikube"
```

Comment:
I suggest a series of test cases for different names,
a for loop to try setting kube context for names with "-" or "_" or containing nunmbers , or myabe even crazier names !